Gustavo.md

```diff
@@ -1,7 +1,26 @@
 # Projeto, Gustavo Souza
 
-Aplicação Web para Sistema de Gerenciamento de Artigos
-A ideia central do projeto é desenvolver um sistema WEB, para que facilite a administração e a pesquisa de artigos pelo usuário, criando uma interface intuitiva, para facilitar a utilização por qualquer tipo de usuário.
+## Introdução
+
+<p>O mundo é movido pelo conhecimento adquirido ao longo dos anos, e existem inúmeras formas de conhecimento para que o ser humano possa adquirir tal feito, e uma das mais conhecidas é através da leitura. Seguindo esse princípio, o surgimento da internet, trouxe consigo as aplicações web, que se tornaram parte de todo esse processo de aprendizado, artigos científicos que antes eram lidos apenas fisicamente, hoje já são integrados ao meio eletrônico.</p>
```

Comment:
@gussouzauni essa primeira frase ficou com dois ` , e ` seguidos. Minha sugestão seria:

`O mundo é movido pelo conhecimento adquirido ao longo dos anos. Existem inúmeras formas de conhecimento para que o ser humano possa adquirir tal feito, uma das mais conhecidas é através da leitura.`

Frases longas são mais difíceis de serem compreendidas.

Reply:
Ok, boa observação.